client/src/pages/Dashboard.js

```diff
@@ -54,6 +61,51 @@ function Dashboard(props) {
     history.push("/dashboard/waitingroom/1");
   }
 
+  const handleFeedbackDialogsClose = (value) => {
+    history.push("/dashboard");
+  };
+
+  const handleOvearllNextQuestionClick = () => {
+    history.push("/dashboard/feedback/2?id=345");
+  };
+
+  const handleReviewDialogNext = () => {
+    history.push("/dashboard/feedback/3?id=777");
+  };
+
+  const handleReviewDialogPrevious = () => {
+    history.push("/dashboard/feedback/1");
+  };
+
+  const handleStrengthsDialogNext = () => {
+    history.push("/dashboard/feedback/4");
+  };
+
+  const handleStrengthsDialogPrevious = () => {
+    history.push("/dashboard/feedback/2");
+  };
+
+  const handleWeaknessesDialogNext = () => {
+    history.push("/dashboard/feedback/5");
+  };
+
+  const handleWeaknessesDialogPrevious = () => {
+    history.push("/dashboard/feedback/3");
+  };
+
+  const handleRecommendationDialogNext = () => {
+    history.push("/dashboard/feedback/6");
+  };
+
+  const handleRecommendationDialogPrevious = () => {
+    history.push("/dashboard/feedback/4");
+  };
```

Comment:
Can this be done more cleanly?

Reply:
What do you suggest?

Reply:
2 functions that dynamically push the history